zproject/computed_settings.py

```diff
@@ -358,34 +360,8 @@
 # REDIS-BASED RATE LIMITING CONFIGURATION
 ########################################################################
 
-RATE_LIMITING_RULES = {
-    "api_by_user": [
-        (60, 200),  # 200 requests max every minute
-    ],
-    "api_by_ip": [
-        (60, 100),
-    ],
-    "api_by_remote_server": [
-        (60, 1000),
-    ],
-    "authenticate_by_username": [
-        (1800, 5),  # 5 failed login attempts within 30 minutes
-    ],
-    "email_change_by_user": [
-        (3600, 2),  # 2 per hour
-        (86400, 5),  # 5 per day
-    ],
-    "password_reset_form_by_email": [
-        (3600, 2),  # 2 reset emails per hour
-        (86400, 5),  # 5 per day
-    ],
-    "sends_email_by_ip": [
-        (86400, 5),
-    ],
-    "spectator_attachment_access_by_file": [
-        (86400, 1000),  # 1000 per day per file
-    ],
-}
+# Merge any local overrides with the default rules.
+RATE_LIMITING_RULES = {**DEFAULT_RATE_LIMITING_RULES, **RATE_LIMITING_RULES}
```

Comment:
This does allow typos in `RATE_LIMITING_RULES` to continue to show up as new domains with `get-django-setting RATE_LIMITING_RULES`.

I don't know if that's going to result in a more confusing, or less confusing, experience than if we stripped them out by only pulling keys that appear in `DEFAULT_RATE_LIMITING_RULES`.

After a brief ponder, I guess that leaving it as-is is probably best -- it will let people see the typo, by dint of seeing both a `api_by_ip` and `api_by_op` key in the output -- as opposed to being confused why their `api_by_ip` isn't kicking in.

Reply:
Yeah, agreed, seems clearer to let the new domain show up